tests/FakeItEasy.IntegrationTests/DummyTests.cs

```diff
@@ -55,8 +55,8 @@ public void Same_dummy_type_can_be_created_concurrently_on_different_threads()
             // the other is still being created.
             // Guards against the concurrent creations being detected as a
             // circular dependency and throwing an exception.
-            var makeOne = Task.Factory.StartNew(A.Dummy<ConcurrentCreationDummy>);
-            var makeTwo = Task.Factory.StartNew(() =>
+            var makeOne = Task.Run(() => A.Dummy<ConcurrentCreationDummy>());
+            var makeTwo = Task.Run(() =>

```

Comment:
I don't remember the warning number, but basically, when using `Task.Factory.StartNew`, a `TaskScheduler` should be specified. In most cases, it's better to use `Task.Run`, unless you want some very specific behavior.